recipes/nm.rb

```diff
@@ -21,6 +22,26 @@
   action :create
 end
 
+template "/etc/sudoers.d/yarn" do
+    source "yarn_sudoers.erb"
+    owner "root"
+    group "root"
+    mode "0440"
+    variables({
+                :user => node["hops"]["yarn"]["user"],
+                :cgroup => "#{node['hops']['base_dir']}/sbin/nm-cgroup-fix.sh"
+              })
+    action :create
+  end
+  
+template "#{node['hops']['home']}/sbin/nm-cgroup-fix.sh" do
+    source "nm-cgroup-fix.sh.erb"
+    owner "root"
+    group node['hops']['group']
+    mode 0544
+end
```

Comment:
Could you please use the kagent provider to setup `sudoers` rules (see the example in `hopsworks::default`)? It allows more control and consistency over the sudoers rules instead of having sudoers file spread across all the cookbooks, like the madness we had before. 

That provider also has a flag to disable setting up sudoers rules across the entire deployment, in case the sudoers rules are managed externally.

Reply:
will do.